public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/DimensionFields.tsx

```diff
@@ -46,11 +62,17 @@ const DimensionFields: React.FC<DimensionFieldsProps> = ({ query, dimensionOptio
               menuShouldPortal
               placeholder="Field"
               value={filter.dimension}
-              options={dimensionOptions}
+              options={validDimensionOptions}
               onChange={(v) => onFieldChange(index, 'dimension', v.value ?? '')}
               width={38}
             />
-            <InlineLabel aria-label="equals">==</InlineLabel>
+            <Select
```

Comment:
optional, but it might be nice to set `==` as the default value

Reply:
This makes sense given that in the backend we default to `==` as well.